backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```diff
@@ -201,9 +206,12 @@ public TestOrder getTestOrder(UUID id) {
 
   @Transactional(readOnly = true)
   public TestOrder getTestOrder(Organization org, UUID id) {
-    return _repo
-        .fetchQueueItemByOrganizationAndId(org, id)
-        .orElseThrow(TestOrderService::noSuchOrderFound);
+    TestOrder order =
+        _repo
+            .fetchQueueItemByOrganizationAndId(org, id)
+            .orElseThrow(TestOrderService::noSuchOrderFound);
+    Hibernate.initialize(order.getResultSet());
```

Comment:
As far as I can tell, this is only required for tests (where things are often called outside of a `@Transactional` context). Would love to hear suggestions on how to make it less...icky

Reply:
I think this is fine. I guess the alternative would be to make `Results` eager load, which is probably not the way